src/pages/careers/locations/index.tsx

```diff
@@ -188,6 +193,16 @@ const query = graphql`
 const Locations = () => {
     const images = useStaticQuery(query)
 
+    const continents = Object.keys(allContinents)
+
+    const formatContinentName = (continent) => {
+        return continent
+            .split('_')
+            .filter((continentName) => continentName.length > 0)
+            .map((continentName) => continentName.charAt(0).toUpperCase() + continentName.slice(1))
```

Comment:
Just this Part, use a method for this
continentName.charAt(0).toUpperCase() + continentName.slice(1))